_replication/replication-methods/log-based-incremental-replication.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Log-based Incremental Replication
+permalink: /replication/replication-methods/log-based-incremental
+keywords: replicate, replication, replication method, stitch replicates data
+tags: [replication]
+layout: general
+category: "settings"
+
+summary: ""
+type: "settings"
+toc: true
+weight: 2
+
+sections:
+  - content: |
+      {% include note.html type="single-line" content="**Note**: Log-based Replication is available only for MySQL and PostgreSQL-backed databases that support binary log replication." %}
+
+      {{ site.data.tooltips.log-based-incremental-rep }} A binary log file is a record of events that occur within a database.
+
+      In this guide, we'll cover:
+
+      1. [How it works (with examples)](#how-log-based-incremental-replication-works),
+      2. [When it should be used](#when-log-based-incremental-replication),
+      3. [Limitations of this Replication Method](#limitations), and
+      4. [How to enable it for your database integration](#enabling-log-based-replication)
+
+  - title: "How {{ page.title }} works"
+    anchor: "how-log-based-incremental-replication-works"
+    content: |
+      There are two types of binary log replication: Statement and row-based. Stitch uses a row-based approach, which means that when rows are modified via [a supported event type](#limitation-2--database-event-types), the entire row is written to the binary log file as a log message.
+
+      During a replication job, Stitch iterates over each message in sequence, or in the same order that the messages were written to the binary log. Data is extracted for any messages that correspond to tables that have been tracked for replication. This means that all tables using {{ page.table }} are extracted from the source database simultaneously as one seamless process.
+
+      To ensure only new and updated data is selected for replication, Stitch will use a binary log's Log Sequence Number (LSN) to 'bookmark' its place in a log file. This value is used to resume replicating data where a previous replication job ended.
+
+    subsections:
+      - title: "Initial replication jobs"
+        anchor: "initial-replication-jobs"
+        content: |
+          During the initial replication job for a table using {{ page.title }}, Stitch will use a `SELECT`-based approach to replicate the table in full. If this were a SQL query, it would look like this:
```

Comment:
should we clarify that we store the MAX LSN that we will return to for log-based replication prior to executing this query?

Reply:
Yeah, I think so, since (I think) that'll be where the second job starts, right? I think I need some clarification on how we determine the max LSN here.

Reply:
Right. I'm not sure what we do to retrieve the max lsn though